source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md

```diff
@@ -0,0 +1,49 @@
+
+- [**Solidus Content**](https://github.com/solidusio-contrib/solidus_content) now supports DatoCMS!
+
+- [**Solidus Subscriptions**](https://github.com/solidusio-contrib/solidus_subscriptions) now integrates with Churn Buster, event emissions and tracking has been improved, and a tab has been added to the admin user view that displays existing subscriptions for any given user. Handy!
+
+- [**Solidus PayPal Commerce Platform**](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform) has had a few minor bugfixes, but more importantly you're now able to display PayPal's credit messaging to your users before checkout, which is shown to increase conversion rates and average order value. Version 0.2.1 is now live!
+
+- [**Solidus PayPal Braintree**](https://github.com/solidusio/solidus_paypal_braintree) has also had PayPal's credit messaging added (when PayPal is being used and the checkout flow is set to `checkout`), in addition to a whole lot of minor improvements and an update to PayPal's new SDK (which allows the new PayPal credit options like Pay in 4 to be available). Version 1.0.0 is now live!
+
+### Core updates this October
+
+Here's what happened in October:
+
+- [Updated permission set admin tab display.](https://github.com/solidusio/solidus/pull/3783) Previously, if a user had access to only a subset of the menu items under 'settings', it would prevent the 'settings' tab from showing up. This bugfix - by Elia Schito - fixes that issue!
+- [Updated incorrect state generation for Italy.](https://github.com/solidusio/solidus/pull/3722) In any instance where a `Carmen::Subregion` existed for a country, we would use the subregion as the countries states. However, in some instances, this is incorrect. In Italys case, we need to be using the subregions subregions. There are more countries with this issue - but this PR provides the framework to fix those countries as they are identified. Sean Denny - that's me - is responsible for this fix!
+- [Added a preference for phone number validation.](https://github.com/solidusio/solidus/pull/3685) Now you can indicate in your config preferences whether or not you'd like to validate phone number presence. This one was also mine!
+- [Updates the install generator to correctly install extra extensions.](https://github.com/solidusio/solidus/pull/3777) There was an issue where, when you're installing extra extensions during Solidus installation (like [`solidus_auth_devise`](https://github.com/solidusio/solidus_auth_devise) or [`solidus_paypal_commerce_platform`](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform)), would fail. This fixes that bug. Another one from me - last one, I swear!
```

Comment:
"would fail" is missing a subject I think. Shouldn't it be "the process would fail" or similar?

Reply:
Good catch, thanks!